### ISSUE TITLE
assertFullyNewStyle is no longer useful

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10277]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)| @wsanchez reported|
|-|-|
|Trac ID|trac#10277|
|Type|enhancement|
|Created|2021-11-17 07:42:59Z|

`src/twisted/internet/test/test_core.py` has an `assertFullyNewStyle` that will never raise in Py3.

See https://github.com/twisted/twisted/pull/1598#pullrequestreview-778224983

<details><summary>Searchable metadata</summary>

```
trac-id__10277 10277
type__enhancement enhancement
reporter__wsanchez wsanchez
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__new new
resolution__None None
component__core core
keywords__None None
time__1637134979278613 1637134979278613
changetime__1640654197521599 1640654197521599
version__None None
owner__wsanchez wsanchez
cc__graingert
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_1"></a>|@wsanchez commented|
|-|-|

https://github.com/twisted/twisted/pull/1673

**Comment by twisted-trac** [comment]
|[<img alt="twm's avatar" src="https://avatars.githubusercontent.com/u/43662?s=50" width="50" height="50">](https://github.com/twm)<a name="note_3"></a>|@twm set owner to @wsanchez|
|-|-|

Adi reviewed. This is good to merge.

**Comment by adiroiban** [comment]
closed in #1673
